src/pages/MainCategory/MainCategory.js

```diff
@@ -1,7 +1,40 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { config } from '../../components/Config/Config';
+
+import './MainCategory.scss';
 
 const MainCategory = () => {
-  return <div></div>;
-};
+  const [category, setCategory] = useState([]);
+  const navigate = useNavigate();
+
+  const goToSubCategory = id => {
+    navigate(`/productlist`);
+  };
 
+  useEffect(() => {
+    fetch(`${config.productApi}/categories/1/subcategories`)
+      .then(res => res.json())
+      .then(data => {
+        setCategory(data.result);
```

Comment:
```suggestion
    fetch(`${API.categories}/1/subcategories`)
      .then(res => res.json())
      .then(data => {
        setCategory(data.result);
```
제가 말씀드린 방법대로 하면 위와 같이 구현할 수 있겠습니다!

Reply:
네 참고하여 수정하겠습니다. 감사합니다!